### PR TITLE
PR Validation: minor changes on top of #17532 (should work now)

### PR DESCRIPTION
**Pull request by ktoso**
Minor changes on top of @2m's #17532 - excluding the benchmarks project, and adding the unidoc settings. Seems to work for me locally, pls build dear validator

### CODE REVIEW

**Comment by ktoso** [comment]
Refs #17532 and adds just a few minor tweaks which should push it ove the finish line

File: project/MultiNode.scala [hunk 1]
```diff
@@ -27,49 +44,65 @@ object MultiNode {
...
+    (executeTests in Test <<= (executeTests in Test, multiTests) map {
+      case (testResults, multiNodeResults)  =>
+        val overall =
+          if (testResults.overall.id < multiNodeResults.overall.id)
+            multiNodeResults.overall
+          else
+            testResults.overall
+        Tests.Output(overall,
+          testResults.events ++ multiNodeResults.events,
+          testResults.summaries ++ multiNodeResults.summaries)
```
Comment (patriknw) [inline project/MultiNode.scala]: great

File: project/package.scala [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package object akka {
+
+  implicit class RichBoolean(val b: Boolean) extends AnyVal {
+    final def apply[A](a: => A): Option[A] = if (b) Some(a) else None
+  }
```
Comment (patriknw) [inline project/package.scala]: do we need this, really?

**Comment by patriknw** [comment]
LGTM

File: project/package.scala (hunk 2 above)
Comment (ktoso) [inline project/package.scala]: I also commented like that :-) It's from the original PR I added my changes on top: https://github.com/akka/akka/pull/17532#discussion_r30734311

**Comment by ktoso** [comment]
Continuing work on @2m's branch on this.

**Pull request closed without merging**